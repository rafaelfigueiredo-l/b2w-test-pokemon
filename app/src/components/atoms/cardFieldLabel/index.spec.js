import CartFieldLabel from "./";

describe("<CartFieldLabel />", () => {
  const image = require("../../../assets/images/pokemonImageNotFound.png");

  const component = <CartFieldLabel>Texto</CartFieldLabel>;

  const mounted = mount(component);
  const wrapper = shallow(component);

  it("should mount correctly CartFieldLabel component", () => {
    expect(wrapper.find(".cartFieldLabel")).toHaveLength(1);
    expect(mounted).toMatchSnapshot();
  });

  it("should unmount correctly CartFieldLabel component", () => {
    expect(mounted.unmount()).toEqual({});
  });

  it("should have a children", () => {
    expect(wrapper.find(".cartFieldLabel").children()).toHaveLength(1);
  });
});
