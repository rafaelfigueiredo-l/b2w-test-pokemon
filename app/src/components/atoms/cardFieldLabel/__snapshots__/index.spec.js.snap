// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CartFieldLabel /> should mount correctly CartFieldLabel component 1`] = `
<styled.label
  className="cartFieldLabel"
>
  <label
    className="sc-AxjAm kizGHW cartFieldLabel"
  >
    Texto
  </label>
</styled.label>
`;
